api/src/main/java/org/apache/iceberg/util/ByteBuffers.java

```diff
@@ -58,6 +59,11 @@ public static ByteBuffer copy(ByteBuffer buffer) {
     return ByteBuffer.wrap(copyArray);
   }
 
+  public static String encodeHexString(ByteBuffer buffer) {
+    byte[] bytes = toByteArray(buffer.duplicate());
```

Comment:
No need for `duplicate` here because `toByteArray` won't modify the buffer.

Reply:
Thanks for your review, I have modified the code